### DISCUSSION TITLE
滤镜 着色器 记录 暂停 字幕 相关问题

### DISCUSSION THREAD

**Discussion by karlmorghulis** [hooke007/mpv_PlayKit#348] [mpv 通用]
之前一直是用的potplayer+madvr+lav，因为最近新换了rtx4070+4k HDR屏，听说mpv对hdr支持比较强就跟着这里设置了三四天了。现在有几个问题我目前没法解决，恳请看到的大佬们帮帮我，谢谢！
①滤镜、着色器这些到底该怎么选用——我常看的是英美剧和电影，几乎不看动漫，请问有合适的推荐一下嘛？
②播放视频时，通过浏览文件目录选择不同的视频文件，再重新选回之前的文件，是可以接着续播的。但是按鼠标侧键进入列表的下一个，再按侧键回去就从起始位置播放了，这个怎么解决呢？
③暂停/停止的状态下，选择另一个视频文件，也是暂停状态，要手动开启播放。可以改成只要是新打开的文件就自动播放嘛？
④播放视频时点击时间轴转跳时间，字幕会接着原时间继续走，直到出现需要新的字幕的时候，才会回到正确的那句字幕。这个怎么解决？

**Comment by hooke007** [comment]
> ③暂停/停止的状态下 ...

`reset-on-next-file=pause`

**Reply by karlmorghulis** [reply]
这个解决了，谢谢大佬。

**Comment by hooke007** [comment]
> ②播放视频时，通过 ...

`playlist-next` 不被视为退出命令，所以不保存进度，也是当前的预期行为。捆绑命令以手动触发保存行为 `write-watch-later-config`

**Reply by karlmorghulis** [reply]
已解决，谢谢大佬！

**Comment by hooke007** [comment]
> ④播放视频时点击时间轴转跳时间 ...

无法复现

**Reply by karlmorghulis** [reply]
再确认了一下，只有sup字幕会这样，srt和ass都不会。
我sup还改了一条相关设置让它输出在画面的黑边上，应该没关系吧。

**Reply by hooke007** [reply]
我印象中只有暴露在外的图形字幕才有这个问题。只要做进封装里不存在这个问题

**Reply by karlmorghulis** [reply]
确实是看的暴露在外的sup字幕，同视频字幕用potplayer播放移动进度条无此问题，所以我以为是可以通过设置解决的问题来着……
那意思是我用mkvtoolnix把字幕封装进去就解决了？

**Reply by hooke007** [reply]
不用封装进视频，封装进容器即可。`mks`

**Reply by karlmorghulis** [reply]
实测可行，感谢大佬。不过一整季的字幕都要逐个封装还是有点蛋疼hhh。这个应该是mpv自身的问题吧，希望以后版本更新能解决……

**Reply by hooke007** [reply]
如果没人反馈的话，也就没人会去解决。
